plugins/content/menu/menu.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.6" type="plugin" group="content" method="upgrade">
+    <name>PLG_CONTENT_MENU</name>
+    <author>Joomla! Project</author>
+    <creationDate>July 2016</creationDate>
+    <copyright>Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
+    <license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+    <authorEmail>admin@joomla.org</authorEmail>
+    <authorUrl>www.joomla.org</authorUrl>
+    <version>1.0.0</version>
+    <description>PLG_CONTENT_MENU_XML_DESCRIPTION</description>
+    <files>
+        <filename plugin="menu">menu.php</filename>
```

Comment:
You haven't include your `form` folder here. It will be problem if you will zip it and install it.